.github/shared/install_sqlite/action.yml

```diff
@@ -0,0 +1,15 @@
+name: "Install SQLite"
```

Comment:
Newest SQLite because the one on CI did not yet support the simplified negative JSON path:

```
sqlite> SELECT '[1,2,3]' -> -1;
3
```